examples/linear_model.ipynb

```diff
@@ -0,0 +1,306 @@
+    "\n",
+    "    Events (2)\n",
+    "        | falling: The object is falling\n",
+    "        | impact: The object has hit the ground\n",
+    "\n",
+    "    Inputs/Loading: (0)\n",
+    "\n",
+    "    States: (2)\n",
+    "        | x: Position in space (m)\n",
+    "        | v: Velocity in space (m/s)\n",
+    "\n",
+    "    Outputs/Measurements: (1)\n",
+    "        | x: Position in space (m)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, for our keyword arguments:\n",
+    "\n",
+    "* `process_noise : Optional, float or dict[str, float]`\n",
+    "  * Process noise (applied at dx/next_state).\n",
+    "  * Can be number (e.g., .2) applied to every state, a dictionary of values for each state (e.g., {'x1': 0.2, 'x2': 0.3}), or a function (x) -> x\n",
+    "* `process_noise_dist : Optional, str`\n",
+    "  * distribution for process noise (e.g., normal, uniform, triangular)\n",
+    "* `measurement_noise : Optional, float or dict[str, float]`\n",
+    "  * Measurement noise (applied in output eqn).\n",
+    "  * Can be number (e.g., .2) applied to every output, a dictionary of values for each\n",
+    "  * output (e.g., {'z1': 0.2, 'z2': 0.3}), or a function (z) -> z\n",
+    "* `measurement_noise_dist : Optional, str`\n",
+    "  * distribution for measurement noise (e.g., normal, uniform, triangular)\n",
+    "* `g : Optional, float`\n",
+    "  * Acceleration due to gravity (m/s^2). Default is 9.81 m/s^2 (standard gravity)\n",
+    "* `thrower_height : Optional, float`\n",
+    "  * Height of the thrower (m). Default is 1.83 m\n",
+    "* `throwing_speed : Optional, float`\n",
+    "  * Speed at which the ball is thrown (m/s). Default is 40 m/s"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With our definitions, we can now create the ThrownObject Model.\n",
+    "\n",
+    "First, we would need to import the necessary packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from prog_models import LinearModel"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll define some features of ThrownObject such that it models after a LinearModel. Remember at the top of the Jupyter Notebook, there were a few properties that all Linear Models must define? In the next step, we'll define our inputs, states, outputs, and events, along with the $A$, $C$, $E$, and $F$ values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(LinearModel):\n",
+    "    events = ['impact']\n",
+    "    inputs = []  \n",
+    "    states = ['x', 'v']\n",
+    "    outputs = ['x']\n",
+    "    \n",
+    "    A = np.array([[0, 1], [0, 0]])\n",
```

Comment:
Let's add some more description of why these are the right values in the markdown

Reply:
I added a more general description of what these values are supposed to represent near the top where we originally defined these values.